### PR TITLE
Documentation: updated slice and join documentation

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+and there is a need to merge these labels into a single, unified label.
+
+![Join masks](/images/joining-tool-01.jpg)
+
+To join masks, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg).
+2. Click on the labels, to select masks that you want to join.
+3. (Optional) To remove selection double click on the mask.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:34]: Click one more time. 
Double click is another mouse event.

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+4. Click again on **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg)
+  (**J**) to execute the join operation.
+
+Upon completion, the selected masks will be merged into a single mask.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:38]:
```suggestion
Upon completion, the selected masks will be joined into a single mask.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+![Join masks gif](/images/joining-tool-02.gif)
+
+## Slicing CVAT labels
+
+The **Slice mask/polygon** (![Slicing tool icon](/images/slicing-tool-icon.jpg))
+is compatible with both mask and polygon annotations.
+
+This tool is useful in scenarios where multiple object in an image
+are annotated with one label labels,
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:48]:
```suggestion
This tool is useful in scenarios where multiple objects in an image
are annotated with one shape,
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+and there is a need to slice this label into multiple labels.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:49]:
```suggestion
and there is a need to slice this shape into multiple parts.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+To slice mask or polygon, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Slice mask/polygon** ![Slicing tool icon](/images/slicing-tool-icon.jpg).
+2. Click on the label you intend to slice.
+   A black contour will appear around the selected label.
+3. Set an initial point for slicing by clicking on the contour.
+4. Draw a line across the label to define the slicing path.
+   <br> **Note**: The line cannot cross itself.
+   <br> **Note**: The line cannot cross the contour more than twice.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:62]: Let's mention right click to cancel the latest point and holding Shift to add points automatically on cursor movement

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+and there is a need to merge these labels into a single, unified label.
+
+![Join masks](/images/joining-tool-01.jpg)
+
+To join masks, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg).
+2. Click on the labels, to select masks that you want to join.
+3. (Optional) To remove selection double click on the mask.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:34]: Why double click?
If you double click you will remove selection and select again

Comment (mdacoca) [reply site/content/en/docs/manual/advanced/slice-and-join.md:34]: Weird... I've updated it but changes are not there....

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+a single label or slicing a single label into several shapes.
+
+This document provides guidance on how to perform these operations effectively.
+
+See:
+
+- [Joining CVAT labels](#joining-cvat-labels)
+- [Slicing CVAT labels](#slicing-cvat-labels)
+
+## Joining CVAT labels
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:18]:
```suggestion
## Joining masks
```

Obviously we speak about CVAT, so, I do not really see a reason to add `CVAT`

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+2. Click on the labels, to select masks that you want to join.
+3. (Optional) To remove the selection click one more time on the mask.
+4. Click again on **Join masks**![Join masks tool icon](/images/join-masks-icon.jpg)
+   (**J**) to execute the join operation.
+
+Upon completion, the selected masks will be joined into a single mask.
+
+![Join masks gif](/images/joining-tool-02.gif)
+
+## Slicing CVAT labels
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:42]:
```suggestion
## Slicing polygons and masks
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+- [Slicing CVAT labels](#slicing-cvat-labels)
+
+## Joining CVAT labels
+
+The **Join masks** tool (![Join masks tool icon](/images/join-masks-icon.jpg)),
+is specifically designed to work with mask annotations.
+
+This tool is useful in scenarios where a single object
+in an image is annotated with multiple shapes,
+and there is a need to merge these shapes into a single, unified shape.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:25]:
```suggestion
and there is a need to merge these shapes into a single one.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+![Join masks](/images/joining-tool-01.jpg)
+
+To join masks, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg).
+2. Click on the labels, to select masks that you want to join.
+3. (Optional) To remove the selection click one more time on the mask.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:34]:
```suggestion
3. (Optional) To remove the selection click the mask one more time.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+in an image is annotated with multiple shapes,
+and there is a need to merge these shapes into a single, unified shape.
+
+![Join masks](/images/joining-tool-01.jpg)
+
+To join masks, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Join masks** ![Join masks tool icon](/images/join-masks-icon.jpg).
+2. Click on the labels, to select masks that you want to join.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:33]:
```suggestion
2. Click on the canvas area, to select masks that you want to join.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+![Join masks gif](/images/joining-tool-02.gif)
+
+## Slicing CVAT labels
+
+The **Slice mask/polygon** (![Slicing tool icon](/images/slicing-tool-icon.jpg))
+is compatible with both mask and polygon annotations.
+
+This tool is useful in scenarios where multiple objects in an image
+are annotated with one shape,
+and there is a need to slice this shape into multiple parts.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:49]: Maybe we should add, that a shape can be sliced only in two parts in time.
Apply the slice tool several times to split a shape to as many parts as you need.

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+![Slicing tool](/images/slicing-tool-01.jpg)
+
+To slice mask or polygon, do the following:
+
+1. From the [**Edit**](/docs/manual/basics/controls-sidebar/#edit) block,
+   select **Slice mask/polygon** ![Slicing tool icon](/images/slicing-tool-icon.jpg).
+2. Click on the label you intend to slice.
+   A black contour will appear around the selected label.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:58]:
```suggestion
2. Click on the shape you intend to slice.
   A black contour will appear around the selected shape.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+3. Set an initial point for slicing by clicking on the contour.
+4. Draw a line across the label to define the slicing path.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/slice-and-join.md:60]:
```suggestion
4. Draw a line across the shape to define the slicing path.
```

File: site/content/en/docs/manual/advanced/slice-and-join.md (hunk 7 above)
Comment (mdacoca) [reply site/content/en/docs/manual/advanced/slice-and-join.md:18]: it's for google search

**Review by bsekachev — APPROVED** [review]

**Pull request merged**